katalog/prometheus-operator/README.md

```diff
@@ -62,7 +63,7 @@ You can deploy Prometheus Operator by running the following command in the root
 the project:
 
 ```shell
-kustomize build | kubectl apply -f -
+kustomize build | kubectl apply -f - --server-side
```

Comment:
the command will not work if you run it in the root of the project as the previous paragraph says, we could either change the `kustomize build` part by adding the right path or changing the text.

PS1: by "root of the project" I understand where the main README is, maybe I'm wrong.

PS2: the same text is present in the rest of the packages' readmes, commenting one time to not spam the PR.

Reply:
I have updated all the READMEs accordingly. :)